cmd/kube-apiserver/app/aggregator.go

```diff
@@ -38,6 +38,7 @@ import (
 	kubeexternalinformers "k8s.io/client-go/informers"
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/kube-aggregator/pkg/apis/apiregistration"
+	apiregistrationscheme "k8s.io/kube-aggregator/pkg/apis/apiregistration/scheme"
```

Comment:
I like to have the scheme separate. But it's not really about apiregistration api. So better move it to pkg/apiserver/scheme in the apiextensions apiserver.

Reply:
ok